app/code/Magento/InventoryAdminUi/Test/Mftf/ActionGroup/UnassignSourceFromProductActionGroup.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="UnassignSourceFromProductActionGroup">
+        <arguments>
+            <argument name="sourceCode" />
```

Comment:
Please specify `type="string"` for this argument.
And also would you mind to add `type="string"` to action group called "AssignSourceToProductActionGroup".